src/GslibIO.jl

```diff
@@ -52,28 +80,51 @@ Optionally set the value used for missings `na`.
 """
 function load_legacy(filename::AbstractString, dims::NTuple{3,Int};
                      origin=(0.,0.,0.), spacing=(1.,1.,1.), na=-999)
-  open(filename) do fs
-    # skip header
-    readline(fs)
-
-    # number of properties
-    nvars = parse.(Int, readline(fs))
+  dataspec = parse_legacy(filename)                     
```

Comment:
I would call this variable simply `spec` to avoid having to type long commands later. `spec` is very clear already.